src/tools/compiletest/src/runtest.rs

```diff
@@ -3559,6 +3559,15 @@ impl<'test> TestCx<'test> {
     }
 
     fn run_rmake_test(&self) {
+        let src_base = self.config.src_base.to_path_buf();
+        if src_base.join("Makefile").exists() {
+            self.run_rmake_legacy_test();
+        } else if src_base.join("rmake.rs").exists() {
+            self.run_rmake_v2_test();
+        }
```

Comment:
Here and in lib.rs you are using different priorities between `Makefile` and `rmake.rs`, which would be confusing if both exist. I think you should error if both exist at the same time.

Reply:
This hasn't been resolved yet, or did I misread the code?

Reply:
You're right, I thought I changed the order but I didn't, addressed in a new commit.